brainscore_language/models/huggingface.py

```diff
@@ -130,16 +130,16 @@ def digest_text(self, text: Union[str, List[str]]) -> Dict[str, DataAssembly]:
     def estimate_reading_times(self, base_output):
         """
         :param base_output: the neural network's output
-        :return: perplexity (calculated as cross entropy) as a proxy for reading times
+        :return: cross entropy as a proxy for reading times, following Smith & Levy 2013
+            (https://www.sciencedirect.com/science/article/pii/S0010027713000413)
         """
         import torch.nn.functional as F
         tokens = self.tokenized_inputs['input_ids'].squeeze()
         logits = base_output.logits.squeeze()
         # assume that reading time is additive,
         # i.e. reading time of multiple tokens is the sum of the perplexity of each individual token
         cross_entropy = F.cross_entropy(logits, tokens, reduction='sum')
```

Comment:
Yes, this would be up to the benchmark. The current implementation of Futrell2018 https://github.com/brain-score/language/blob/55a0793d8c1e00eab9d08ab5217009ba22026c03/brainscore_language/plugins/futrell2018/benchmark.py#L42-L43 goes word-by-word. Would you explicitly guard against this from the model side?

Reply:
from zoom call: don't guard against it, but document clearly